quick-build.sh

```diff
@@ -1,41 +1,6 @@
 #!/bin/sh
 
-export MAVEN_OPTS='-Xmx256m'
-
-echo ''
-echo ''
-echo '                 uuuuuuu'
-echo '             uu$$$$$$$$$$$uu'
-echo '          uu$$$$$$$$$$$$$$$$$uu'
-echo '         u$$$$$$$$$$$$$$$$$$$$$u'
-echo '        u$$$$$$$$$$$$$$$$$$$$$$$u'
-echo '       u$$$$$$$$$$$$$$$$$$$$$$$$$u'
-echo '       u$$$$$$$$$$$$$$$$$$$$$$$$$u'
-echo '       u$$$$$$"   "$$$"   "$$$$$$u'
-echo '       "$$$$"      u$u       $$$$"'
-echo '        $$$u       u$u       u$$$'
-echo '        $$$u      u$$$u      u$$$'
-echo '         "$$$$uu$$$   $$$uu$$$$"'
-echo '          "$$$$$$$"   "$$$$$$$"'
-echo '            u$$$$$$$u$$$$$$$u'
-echo '             u$"$"$"$"$"$"$u'
-echo '  uuu        $$u$ $ $ $ $u$$       uuu'
-echo ' u$$$$        $$$$$u$u$u$$$       u$$$$'
-echo '  $$$$$uu      "$$$$$$$$$"     uu$$$$$$'
-echo 'u$$$$$$$$$$$uu    """""    uuuu$$$$$$$$$$'
-echo '$$$$"""$$$$$$$$$$uuu   uu$$$$$$$$$"""$$$"'
-echo ' """      ""$$$$$$$$$$$uu ""$"""'
-echo '           uuuu ""$$$$$$$$$$uuu'
-echo '  u$$$uuu$$$$$$$$$uu ""$$$$$$$$$$$uuu$$$'
-echo '  $$$$$$$$$$""""           ""$$$$$$$$$$$"'
-echo '   "$$$$$"                      ""$$$$""'
-echo '     $$$"                         $$$$"'
-echo ''
-echo ''
-echo '           TESTS ARE DISABLED'
-echo '               ARE U CRAZY ?'
-echo ''
-echo ''
-echo ''
-
-mvn clean install -e -DskipTests -Pdev $*
+# Parallel executions of maven modules and tests.
+# Half of CPU core are used in to keep other half for OS and other programs.
+# As long as web JS tests are not stable, it's recommended to execute: quick-build.sh -DskipWebTests=true
+mvn clean install -T0.5C -DforkCount=0.5C $*
```

Comment:
quick-build.sh was unsafe-build.sh.
But in fact a build should always be fast, so I renamed quick-build.sh to build.sh and dropped unsafe-build.sh